source/images/download-full-reverse.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="44px" height="44px" viewBox="0 0 44 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <title>Download Link</title>
+    <desc>A two-part list of links to download the article, or parts of the article, in various formats.</desc>
```

Comment:
This isn't true for cases where it downloads a podcast episode, for example.

Reply:
These elements should describe the image, not its context, which the image can't know. `<desc>` is optional and probably not needed.